types/dragino/groPointAirV1/uplink.js

```diff
@@ -0,0 +1,86 @@
+function consume(event) {
+  const bytes = Hex.hexToBytes(event.data.payloadHex);
+  const moisture = {};
+  const temperature = {};
+  const lifecycle = {};
+
+  const tem = [
+    "temperature1",
+    "temperature2",
+    "temperature3",
+    "temperature4",
+    "temperature5",
+    "temperature6",
+    "temperature7",
+
+    "temperature8",
+    "temperature9",
+    "temperature10",
+    "temperature11",
+    "temperature12",
+    "temperature13",
+  ];
+
+  const hum = [
+    "soilMoisture1",
+    "soilMoisture2",
+    "soilMoisture3",
+    "soilMoisture4",
+    "soilMoisture5",
+    "soilMoisture6",
+    "soilMoisture7",
+    "soilMoisture8",
+  ];
+
+  const temLen = [0, 0, 4, 6, 7, 9, 11, 12, 13];
```

Comment:
use more descriptive variable names throughout this file

Reply:
Its a decoder without explanation from the sensor manufacturer.